docs/reference/ml/df-analytics/apis/evaluate-dfanalytics.asciidoc

```diff
@@ -42,31 +42,32 @@ result field to be present.
 [[ml-evaluate-dfanalytics-request-body]]
 ==== {api-request-body-title}
 
+`index`::
```

Comment:
Done.

I'm wondering what to do with the order of `actual_field`, `predicted_field` and `metrics`. Alphabetical order would be `actual_field`, `metrics`, `predicted_field` but logically, `actual_field` and `predicted_field` are related and it would be good to read them together.

@szabosteve: WDYT?

Reply:
@przemekwitek 
Yeah, I was on the fence with this one.
As this section contains only three main objects that can be easily overviewed, it is okay to keep the logical order. It's better for the user because of the reason you mentioned.
Deal?

Reply:
Deal :)